modules/core/package.json

```diff
@@ -25,9 +25,7 @@
     "dist.min.js"
   ],
   "sideEffects": [
-    "dist/es5/lib/init.js",
-    "dist/esm/lib/init.js",
-    "dist/es6/lib/init.js"
+    "**/init.js"
```

Comment:
In my experience (possibly incorrect) directory names needed to start with './' to work. Perhaps to distinguish them from modules. But I assume that is not needed for wildcards.

Minor Nit: I am on the fence about using a wildcard to cover both src and dist. I would not realize that the wildcard covers both trees at first glance.

Maybe 
```
./dist/**/init.js'
./src/init.js
```
Would make it a little clearer what is going on.